package.json

```diff
@@ -19,62 +19,62 @@
   },
   "dependencies": {
     "apollo-link-retry": "^2.2.16",
-    "bufferutil": "^4.0.7",
-    "loglevel": "^1.8.0",
-    "plotly.js": "^2.20.0",
-    "plotly.js-basic-dist": "^2.20.0",
+    "bufferutil": "^4.0.8",
+    "loglevel": "^1.9.2",
+    "plotly.js": "^2.35.2",
+    "plotly.js-basic-dist": "^2.35.2",
     "react-plotly.js": "^2.6.0",
-    "redux": "^4.1.2",
+    "redux": "^4.2.1",
     "utf-8-validate": "^5.0.10",
     "uuid": "^9.0.1"
   },
   "devDependencies": {
-    "@apollo/client": "^3.6.0",
-    "@material-ui/core": "^4.9.9",
-    "@rollup/plugin-commonjs": "^21.0.1",
-    "@rollup/plugin-node-resolve": "^13.1.3",
-    "@rollup/plugin-typescript": "^8.3.0",
-    "@testing-library/jest-dom": "^5.16.2",
-    "@testing-library/react": "^12.0.0",
-    "@types/jest": "^27.4.0",
-    "@types/node": "^22.5.2",
-    "@types/react-plotly.js": "^2.5.2",
-    "@types/react-test-renderer": "^17.0.1",
+    "@apollo/client": "^3.11.8",
+    "@emotion/styled": "^11.13.0",
+    "@mui/material": "^6.1.6",
+    "@rollup/plugin-commonjs": "^28",
+    "@rollup/plugin-node-resolve": "^15",
+    "@rollup/plugin-typescript": "^12",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^16.0.1",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^22.9.0",
+    "@types/react-plotly.js": "^2.6.3",
+    "@types/react-test-renderer": "^18.3.0",
     "@types/uuid": "^9.0.8",
-    "@vitejs/plugin-react": "^4.3.1",
-    "base64-js": "^1.3.1",
+    "@typescript-eslint/typescript-estree": "^8.13.0",
+    "@vitejs/plugin-react": "^4.3.3",
+    "base64-js": "^1.5.1",
     "clipboard-copy": "^4.0.1",
     "eslint-config-prettier": "^9.1.0",
     "eslint-config-react-app": "^7.0.1",
-    "eslint-plugin-import": "^2.30.0",
+    "eslint-plugin-import": "^2.31.0",
     "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-unicorn": "^55.0.0",
-    "graphql-ws": "^5.9.1",
-    "jsdom": "^25.0.0",
-    "loglevel": "^1.8.0",
+    "graphql-ws": "^5.16.0",
+    "jsdom": "^25.0.1",
+    "loglevel": "^1.9.2",
     "prettier": "^3.3.3",
-    "react-id-generator": "^3.0.0",
-    "react-test-renderer": "^17.0.0",
-    "react-tiny-popover": "^6.0.10",
-    "rollup": "^2.63.0",
-    "rollup-plugin-dts": "^4.1.0",
-    "rollup-plugin-peer-deps-external": "^2.2.4",
-    "rollup-plugin-postcss": "^4.0.2",
```

Comment:
When attempting to build with `npm run rollup`, I received errors that these packages are missing. They're used in rollup.config.js.